zilencer/tests/test_stripe.py

```diff
@@ -198,6 +205,19 @@ def test_upgrade_with_outdated_seat_count(self, mock_create_subscription: mock.M
             event_type=RealmAuditLog.PLAN_UPDATE_QUANTITY).values_list('extra_data', flat=True).first()),
             {'quantity': new_seat_count})
 
+    @mock.patch("zilencer.lib.stripe.STRIPE_PUBLISHABLE_KEY", "stripe_publishable_key")
+    @mock.patch("zilencer.views.STRIPE_PUBLISHABLE_KEY", "stripe_publishable_key")
+    def test_upgrade_with_invalid_salt(self) -> None:
+        self.login(self.user.email)
+        post_data = {
```

Comment:
I think these two lines can be combined into one; we don't generally assign variable names to things that are only used once.